pkg/printer/printer.go

```diff
@@ -67,6 +76,10 @@ func NewPrinter(f string, w io.Writer) (Printer, error) {
 		p = NewJSON(w)
 	case OutFormatSonarQube:
 		p = NewSonarQube(w)
+	case OutFormatPrometheus:
+		p = NewPrometheus(w, c)
```

Comment:
Ideally you do not need to pass the entire config into these two New* functions, you should only need to pass in what is needed for the printer to run.